src/Files.App/Services/PreviewPopupProviders/SeerProProvider.cs

```diff
@@ -43,5 +51,58 @@ public async Task<bool> DetectAvailability()
 			var handle = User32.FindWindow("SeerWindowClass", null).DangerousGetHandle();
 			return handle != IntPtr.Zero && handle.ToInt64() != -1;
 		}
+
+		private bool? _IsTrackSelectionSettingEnabledCache;
+		private bool IsTrackSelectionSettingEnabled
+		{
+			get
+			{
+				if (_IsTrackSelectionSettingEnabledCache is null)
+					_IsTrackSelectionSettingEnabledCache = DetectTrackSelectionSetting().Result;
+
+				return _IsTrackSelectionSettingEnabledCache.Value;
+			}
+		}
+
+		private Task<bool> DetectTrackSelectionSetting()
+		{
+			bool trackSelectedFile = true;
+
+			// List of possible paths for the Seer Pro settings file
+			string[] paths =
+			{
+				Environment.ExpandEnvironmentVariables("%USERPROFILE%\\Documents\\Seer\\uwp.ini"),
```

Comment:
It might help us with other providers if we create a constant class 'Constants.CloudDriveSettings' or something similar one.

Reply:
This is only for SeerPro, other providers don't have this option.

Reply:
Spreading hard coded strings doesn't look great however this issue is out of scope and so I'm fine for now.

Reply:
Okay